tests/test_decryption_mediator.py

```diff
@@ -205,6 +205,9 @@ def test_announce(self):
         # Cannot get plaintext tally without a quorum
         self.assertIsNone(subject.get_plaintext_tally())
 
+    def tearDown(self):
+        self.key_ceremony.reset(CeremonyDetails(self.NUMBER_OF_GUARDIANS, self.QUORUM))
+
```

Comment:
![](https://media.giphy.com/media/pEP4CwhG1f1uw/giphy.gif)

This caused an incredible amount of stress.